plugins/s3_auth/aws_auth_v4.cc

```diff
@@ -29,7 +29,6 @@
 #include <openssl/sha.h>  /* SHA(), sha256_Update(), SHA256_Final, etc. */
 #include <openssl/hmac.h> /* HMAC() */
 
-#undef AWS_AUTH_V4_DETAILED_DEBUG_OUTPUT
```

Comment:
@shinrich I am removing the `#undef` (same end result), used it to manually enable/disable some extra debugging info sent to `stdout` during development (not needed during the normal use in the plugin or in the unit-tests).